db/migrate/20150625164705_make_merged_at_datetime.rb

```diff
@@ -0,0 +1,6 @@
+class MakeMergedAtDatetime < ActiveRecord::Migration
+  def change
+    remove_column :hubstats_pull_requests, :merged_at
+    add_column :hubstats_pull_requests, :merged_at, :datetime
```

Comment:
Did you test saving a string date value from GH to this new column?  Does it automatically convert the type or do we also need to add some conversion logic to the pull request model?